frontend/src/container/WorkspaceContent.jsx

```diff
@@ -513,7 +513,7 @@ export class WorkspaceContent extends React.Component {
 
     const fetchMoveContent = await props.dispatch(putContentItemMove(source, destination))
     if (fetchMoveContent.status !== 200) {
-      switch (fetchMoveContent.json.code) {
+      switch ((fetchMoveContent.json || {}).code) {
```

Comment:
you should add a property `code` to your default value

Reply:
not needed, it will just be undefined. Do you prefer that I add a key anyway?

Reply:
done